src/mantarray_waveform_analysis/peak_detection.py

```diff
@@ -88,14 +88,13 @@ def peak_detector(
         distance=minimum_required_samples_between_twitches,
         prominence=max_prominence / 4,
     )
-    left_valley_bases = properties["left_bases"]
-    right_valley_bases = properties["right_bases"]
+    left_ips = properties["left_ips"]
+    right_ips = properties["right_ips"]
+
     # TODO Tanner (11/3/20): move this loop to find_twitch_indices
+    # Patches error in B6 file for when two valleys are found in a single valley. If this is true left_bases, right_bases, prominences, and raw magnetic sensor data will also be equivalent to their previous value. This if statement indicates that the valley should be disregarded if the interpolated values on left and right intersection points of a horizontal line at the an evaluation height are equivalent. This would mean that the left and right sides of the peak and its neighbor peak align, indicating that it just one peak rather than two.
```

Comment:
can you include a hyperlink to the scipy docs about what `ips` is?

Also, how is it decided which of the two potential peaks is the "true" peak, and which one is "bad" and should be deleted?  it seems arbitrary to me at the moment, but maybe I'm missing something.

also, please bump minor version number for release so that once this patch is in place we'll be able to incorporate into SDK. since it's a release, it will need some minor description in the changelog also

Reply:
![Screen Shot 2021-02-10 at 7 51 37 PM](https://user-images.githubusercontent.com/69088028/107600760-88ac4a80-6bd9-11eb-8f29-5b0711144226.png)

Reply:
This is Brandon's file when both peaks are found. As of now the patch arbitrarily removes the second detected peak.

Reply:
It seems like it might make sense to choose the highest peak instead of arbitrarily the 1st or 2nd one. That seems like a better way to determine the "true" peak. What do you think?

Reply:
yes I think so too. In this case unless we give it height as a kwarg then we can't have the height values for each peak. would it make sense to use the prominence values instead or supply the find_peaks function with a height value?